docs/source/tutorial/transform.rst

```diff
@@ -36,81 +36,12 @@ The ``transform_function`` method has only one argument, the ``mapped_df_dict``
 which is a dict of extract config file paths and the corresponding Pandas
 DataFrames produced by the extract configs.
 
-As you can see, right now the transform module is very simple. It doesn't
-really merge anything. It simply returns one of the extract stage DataFrames.
-
-Transform Function
-==================
-
-Let's modify the transform function so that it merges our two extracted
-DataFrames (corresponding to `data/clinical.tsv` and
-`data/family_and_phenotype.tsv`).
-
-We'll explain the pieces, but this is the end result:
-
-.. code-block:: python
-
-    """
-    Auto-generated transform module
-
-    Replace the contents of transform_function with your own code
-
-    See documentation at
-    https://kids-first.github.io/kf-lib-data-ingest/ for information on
-    implementing transform_function.
-    """
-    import os
-
-    # Use these merge funcs, not pandas.merge
-    from kf_lib_data_ingest.common.pandas_utils import (
-        outer_merge,
-        merge_wo_duplicates
-    )
-    from kf_lib_data_ingest.common.concept_schema import CONCEPT
-    from kf_lib_data_ingest.config import DEFAULT_KEY
-
-    def transform_function(mapped_df_dict):
-        dfs = {
-            os.path.basename(fp): df
-            for fp, df in
-            mapped_df_dict.items()
-        }
-        clinical_df = dfs['extract_config.py']
-        family_and_phenotype_df = dfs['family_and_phenotype.py']
-
-        merged = outer_merge(clinical_df,
-                             family_and_phenotype_df,
-                             on=CONCEPT.PARTICIPANT.ID,
-                             with_merge_detail_dfs=False)
-
-        return {
-            DEFAULT_KEY: merged
-        }
-
-First we just reshape the input data so that it's easier to work with:
-
-.. code-block:: python
-
-    dfs = {
-        os.path.basename(fp): df
-        for fp, df in
-        mapped_df_dict.items()
-    }
-    clinical_df = dfs['extract_config.py']
-    family_and_phenotype_df = dfs['family_and_phenotype.py']
-
-Next, we merge the two DataFrames together on the participant ID column.
-
-.. code-block:: python
-
-    merged = outer_merge(clinical_df,
-                         family_and_phenotype_df,
-                         on=CONCEPT.PARTICIPANT.ID,
-                         with_merge_detail_dfs=False)
+Modify the transform function so that it merges your extracted DataFrames
+appropriately according to the function docstring for transform_function above.
 
 There are a few important things to note in this step:
 
-1. We outer merged the DataFrames
+1. We outer merged DataFrames so that we don't lose data
```

Comment:
These pointers don’t make sense now because they’re talking about a section that was removed. 

Why are we removing the transform function code? I think the entire point of the tutorial is to show step by step how to develop a working ingest package. Both extract stage and transform stage tutorial sections should result in working code. Shouldn’t we still write out a working transform function here?

Reply:
> Why are we removing the transform function code?

Because you can't say to test after the extract section and also require code that only works after the transform section.